source/hkdf.c

```diff
@@ -77,14 +77,14 @@ static int aws_cryptosdk_hkdf_expand(
     if (!prk || !okm->len || !prk_len) goto err;
     n = (okm->len + hash_len - 1) / hash_len;
     if (n > 255) goto err;
-
-    for (uint8_t idx = 1; idx <= n; idx++) {
+    for (uint32_t idx = 1; idx <= n; idx++) {
+        uint8_t idx_local = idx;
```

Comment:
Anyone reading this can see it is a local variable. I would probably give it a name that is more descriptive of what it is doing in the algorithm, such as ``idx_byte``. Not blocking on this.